tests/publiccloud/prepare_instance.pm

```diff
@@ -8,7 +8,7 @@
 # without any warranty.
 
 # Package: openssh
-# Summary: This tests will deploy the public cloud instance and prepare the ssh
```

Comment:
this shouldnt be removed IMO. a preparation of ssh is happening if you want to be accurate.

Reply:
Review now please.

Reply:
betta :)